README.md

```diff
@@ -247,4 +247,4 @@ a Rust-native cloud development platform that allows deploying Serenity bots for
 [repo:lavaplayer]: https://github.com/sedmelluq/lavaplayer
 [logo]: https://raw.githubusercontent.com/serenity-rs/serenity/current/logo.png
 [rust-version-badge]: https://img.shields.io/badge/rust-1.64.0+-93450a.svg?style=flat-square
-[rust-version-link]: https://blog.rust-lang.org/2022/09/22/Rust-1.64.0.html
+[rust-version-link]: https://blog.rust-lang.org/2022/09/22/Rust-1.64.0.html
```

Comment:
Still a spurious unrelated change here. Probably just a trailing newline added/removed

Reply:
should be gone now